core-web/libs/portlets/dot-ema/src/lib/components/edit-ema-language-selector/edit-ema-language-selector.component.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { Spectator, byTestId, createComponentFactory } from '@ngneat/spectator';
+import { of } from 'rxjs';
+
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { By } from '@angular/platform-browser';
+
+import { DotLanguagesService } from '@dotcms/data-access';
+import { DotLanguagesServiceMock } from '@dotcms/utils-testing';
+
+import { EmaLanguageSelectorComponent } from './edit-ema-language-selector.component';
+
+import { EditEmaStore } from '../../feature/store/dot-ema.store';
+import { DotPageApiService } from '../../services/dot-page-api.service';
+
+describe('DotEmaLanguageSelectorComponent', () => {
+    let spectator: Spectator<EmaLanguageSelectorComponent>;
+    let store: EditEmaStore;
+    let component: EmaLanguageSelectorComponent;
+
+    const createComponent = createComponentFactory({
+        component: EmaLanguageSelectorComponent,
+        imports: [HttpClientTestingModule],
+        providers: [
+            EditEmaStore,
+            {
+                provide: DotPageApiService,
```

Comment:
Don't need this... this component don't care where their inputs come from, just add them in the factory.